### PR TITLE
[feat] Added issues templates

### LINKED ISSUE #3: Create Issue templates
A few basic issue templates are needed. You can refer to these- https://github.com/avneesh0612/.github/tree/main/.github/ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by 0xMukesh**
## 🛠️ Fixes Issue

Closes #3 

## 👨‍💻 Changes proposed

I have added issue templates for bug reports, feature request and docs.

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    validations:
+      required: false    
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional information
+      description: Add any other information about the problem here.
+    validations:
+      required: false 
+
```
Comment (suitupstan) [inline .github/ISSUE_TEMPLATE/bug.yml]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/docs.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      description: Please add screenshots if applicable
+    validations:
+      required: false
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this issue?
+    validations:
+      required: false
```
Comment (suitupstan) [inline .github/ISSUE_TEMPLATE/docs.yml]:
```suggestion
      required: false

```

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    validations:
+      required: false    
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional information
+      description: Add any other information about the problem here.
+    validations:
+      required: false 
+
```
Comment (suitupstan) [inline .github/ISSUE_TEMPLATE/feature.yml]:
```suggestion

```

File: .github/ISSUE_TEMPLATE/other.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      description: Provide a clear and concise explanation of your issue.
+    validations:
+      required: true
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this issue?
+    validations:
+      required: false
```
Comment (suitupstan) [inline .github/ISSUE_TEMPLATE/other.yml]:
```suggestion
      required: false

```

File: .github/ISSUE_TEMPLATE/docs.yml (hunk 2 above)
Comment (0xMukesh) [reply .github/ISSUE_TEMPLATE/docs.yml]: 🤔 Hey could I know what's the issue over here ?

File: .github/ISSUE_TEMPLATE/other.yml (hunk 4 above)
Comment (0xMukesh) [reply .github/ISSUE_TEMPLATE/other.yml]: 🤔 Hey could I know what's the issue over here ?

File: .github/ISSUE_TEMPLATE/docs.yml (hunk 2 above)
Comment (suitupstan) [reply .github/ISSUE_TEMPLATE/docs.yml]: End line

File: .github/ISSUE_TEMPLATE/other.yml (hunk 4 above)
Comment (suitupstan) [reply .github/ISSUE_TEMPLATE/other.yml]: End line

Comment (0xMukesh) [reply .github/ISSUE_TEMPLATE/other.yml]: > 

Is that for formatting or is it required ? 🤔 
I am new to making these issue templates via `.yml`, so just wanted to know

**Review by suitupstan — APPROVED** [review]

**Pull request merged**